### PR TITLE
Cr51

### PR DESCRIPTION
**Pull request by Aurige**
Dynamic Stop Assignment extension to link the STOP ASSIGNMENT to a JOURNEY PATTERN or a VEHICLE JOURNEY

### CODE REVIEW

**Comment by Aurige** [comment]
Small update to solve 2 small issues linked to CR51

File: xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd [hunk 1]
```diff
@@ -550,4 +551,82 @@ Rail transport, Roads and Road transport
...
+		<xsd:annotation>
+			<xsd:documentation>Elements for a VEHICLE JOURNEY STOP ASSIGNMENT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element ref="VehicleJourneyRef" minOccurs="0" maxOccurs="unbounded"/>
+			<xsd:element ref="PassengerStopAssignmentRef" minOccurs="0"/>
+		</xsd:sequence>
+	</xsd:group>
+	
+	<xsd:complexType name="VehicleJourneyStopAssignmentsInFrame_RelStructure">
```
Comment (skinkie) [inline xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd:614]: It seems that most of this change is already in the current master. But VehicleJourneyStopAssignmentsInFrame_RelStructure is not.

Comment (Aurige) [reply xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd:614]: So what is the best way to go from here: shall we close this one and creat a new PR or will Merge be able to manage this ? (we obviusly need to be able to carry this info in a Frame)

Comment (skinkie) [reply xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd:614]: The foremost problem is there is zero documentation other than mentioning of the change request.

Comment (Aurige) [reply xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourney_version.xsd:614]: Right... I just added the "needs documentation update" label

File: xsd/netex_part_1/part1_tacticalPlanning/netex_stopAssignment_version.xsd [hunk 2]
```diff
@@ -263,7 +263,7 @@ Rail transport, Roads and Road transport
 				<xsd:sequence>
 					<xsd:group ref="DynamicStopAssignmentGroup">
 						<xsd:annotation>
-							<xsd:documentation>Elements for a DYNAMIC PASSENGER STOP ASSIGNMENT.  Original Passenger Stop Assignment for whiuch this is a change.</xsd:documentation>
+							<xsd:documentation>Elements for a DYNAMIC PASSENGER STOP ASSIGNMENT.  Original Passenger STop Assignment for which this is a change.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_tacticalPlanning/netex_stopAssignment_version.xsd:266]:
```suggestion
							<xsd:documentation>Elements for a DYNAMIC PASSENGER STOP ASSIGNMENT.  Original PASSENGER STOP ASSIGNMENT for which this is a change.</xsd:documentation>
```

**Review by skinkie — APPROVED** [review]

**Comment by skinkie** [comment]
Sundays are the best Mondays.

**Comment by ue71603** [comment]
I suggest closing this one in favor of: https://github.com/NeTEx-CEN/NeTEx/pull/343
@Aurige

**Review by ue71603 — COMMENTED** [review]
everything is already in master. ==> close

**Comment by Aurige** [comment]
Already in PArt2 doc

**Pull request closed without merging**